stdlib/3/typing.pyi

```diff
@@ -284,7 +284,7 @@ class IO(Iterable[AnyStr], Generic[AnyStr]):
     @abstractmethod
     def __enter__(self) -> 'IO[AnyStr]': ...
     @abstractmethod
-    def __exit__(self, type, value, traceback) -> bool: ...
+    def __exit__(self, t: type, value: Any, traceback: Any) -> bool: ...
```

Comment:
Same thing as before -- t is optional and value should be an `Optional[BaseException]`, I think.